src/sci/impl/protocols.cljc

```diff
@@ -195,18 +208,18 @@
         expansion
         `(do
            ~@(map (fn [[type & meths]]
-                    `(do
-                       (clojure.core/alter-var-root
-                        (var ~protocol-name) update :satisfies (fnil conj #{})
-                        (symbol (str ~type)))
-                       ~@(process-methods ctx type meths pns extend-via-metadata)))
+                    (let [type #?(:clj type
+                                  :cljs (get cljs-type-symbols type type))]
+                      `(do
+                         (clojure.core/alter-var-root
+                          (var ~protocol-name) update :satisfies (fnil conj #{})
+                          (symbol (str ~type)))
```

Comment:
above this line I do a mapping in CLJS from `'default` to a specific string (see `extend-default-val`). the reason being that if I leave the symbol as is, this line will evaluate a lookup of the var `default` and find it missing, throwing an error.

using the special sentinel string seems kind of gross, but it makes the code tidy here as all I need to do is add the entry to the `cljs-type-symbols` map I need for the other type symbols, and update `default?`